### PR TITLE
Clarify units of numeric values

### PR DESCRIPTION
**Pull request by ykulbak**
Following https://github.com/buildingSMART/BCF-XML/issues/353 
and 
https://github.com/buildingSMART/BCF-XML/pull/357

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ The Open CDE workgroup develops the BCF standard. The group meets every second M
   * [1.2 Authorization](#12-authorization)
     + [1.2.1 Per-Entity Authorization](#121-per-entity-authorization)
     + [1.2.2 Determining Authorized Entity Actions](#122-determining-authorized-entity-actions)
+  * [1.3 Units of numeric values](#13-units-of-numeric-values)
```
Comment (GeorgDangl) [inline README.md:34]:
```suggestion
  * [1.3 Units of Numeric Values](#13-units-of-numeric-values)
```

File: README.md [hunk 2]
```diff
@@ -191,6 +192,10 @@ Indicating that for this topic, the current user can:
 * place the Topic into `closed` status
 * leave the Topic `open` or place the topic back into `open` status after closing it
 
+### 1.3 Units of numeric values
```
Comment (GeorgDangl) [inline README.md:195]:
```suggestion
### 1.3 Units of Numeric Values
```

**Review by GeorgDangl — CHANGES_REQUESTED** [review]
Just switched to Title Case here😀

**Comment by ykulbak** [comment]
@GeorgDangl thanks. I've accepted your changes. Please squash before merging.

**Review by GeorgDangl — APPROVED** [review]

**Pull request merged**